opendihu_examples/isochoric_contraction/optimize_prestretch_length/cuboid_muscle/README.md

```diff
@@ -0,0 +1,16 @@
+# Bayesian Optimization for a cuboid muscle model
+
+## Setup
+- A dummy cuboid muscle geometry. 
+- The solvers for both stretching and contraction are coupled mechanics solver and fastmonodomain solver. In the prestretch process we set dynamic to `False` and add boundary conditions that simulate the muscle being fixed at one side and being pulled at from the other side until it has reached a given length. In the contraction process we set dynamic to `True` and let the ends of the muscel free.
+- It uses the electrophysiology CellML model "hodgkin_huxley-razumova" and the incompressible mechanics model "Mooney-Rivlin".
+- No preCICE involved. 
+
+## How to run
+To run a single simulation of stretching and contracting a muscle, go to build_release and run:
+```
+./muscle_contraction_with_prestretch ../settings_contraction_with_prestretch.py incompressible_mooney_rivlin 10.0
+```
+
+## Optimization
+With this case we can use Bayesian Optimization to optimize the contraction length. Our function f: R -> R maps a prestretch length to the contraction length of a muscle, that has been stretched until it has reached the prestretch length before contracting. One function evaluation is one simulation of the muscle. This way the optimization process outputs the prestretch length that leads to the greatest contraction length of our given muscle.
```

Comment:
Again I feel like this needs more explanation. 
- What's the prestretch length? Total length or the difference?
- How do you define contraction length? Do you look at the average z position of both ends and substract it?